fubectl.source

```diff
@@ -2,13 +2,15 @@
 alias _inline_fzf="fzf --multi --ansi -i -1 --height=50% --reverse -0 --header-lines=1 --inline-info --border"
 alias _inline_fzf_nh="fzf --multi --ansi -i -1 --height=50% --reverse -0 --inline-info --border"
 
-_isClusterSpaceObject() {
+_isObjectSpecified(){
```

Comment:
Quite some added "complexity" for a simple `[ -z "$foo" ] && echo "please specify foo"`
Also the name does not indicate that the main purpose of the function is the log message.